### PR TITLE
Read 128 bit trace ID in End Invocation

### PR DESCRIPTION
**Pull request by agocs**
### What does this PR do?

@duncanista pointed out we also need to read the upper 64 bits of a trace ID in the End Invocation flow. This is a quick hack to add that. The _right_ way, of course, would be using `tracer.Inject` and `tracer.Extract`.

### Describe how to test/QA your changes

I created an AWS Step Function and Java, .Net, and Go lambda functions. The step function invokes the lambda functions. I instrumented the Lambda functions with the Lambda Extension and  Universal Instrumentation. I then invoked the step function and verified that the Lambda functions correctly read the distributed trace context from the step function and passed that trace context back and forth between the Extension and the tracer.

image

### CODE REVIEW

**Review by duncanista — APPROVED** [review]

File: releasenotes/notes/read-128-bit-trace-id-in-end-invocation-9d44a9373f4f712d.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+enhancements:
+  - |
+    The AWS Lambda Extension is now  able to read the full 128 bit trace ID
```
Comment (buraizu) [inline releasenotes/notes/read-128-bit-trace-id-in-end-invocation-9d44a9373f4f712d.yaml:4]:
```suggestion
    The AWS Lambda Extension is now  able to read the full 128-bit trace ID 
```

**Review by buraizu — APPROVED** [review]
Approving with a minor update requested

**Pull request merged**